tasks/agent.py

```diff
@@ -231,7 +231,7 @@ def omnibus_build(ctx, puppy=False, log_level="info", base_dir=None, gem_path=No
             "log_level": log_level,
             "overrides": overrides_cmd
         }
-        env = {}
+        env = load_release_versions(ctx, os.environ.get("RELEASE_VERSION", "nightly"))
```

Comment:
just a nit: I think it'd make more sense to have `release_version` as an argument of the `omnibus_build` command rather than pulling the value directly as an env var

Reply:
👍